### PR TITLE
 Fix #1236 ,#1234 and code refactoring

### LINKED ISSUE #1236: [Status Dashboard] At component recent metrics values are round off to the intergers
**Description:**
At component, recent metrics values are round off to the integers when calculating latency and throughput. So always it displays as zero. 

screenshot from 2018-05-07 02-13-03

**Suggested Labels:**
Bug

**Suggested Assignees:**
@Yasara123

### PR DESCRIPTION
**Pull request by Yasara123**

### CODE REVIEW

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/QueryManager.java [hunk 1]
```diff
@@ -40,10 +38,10 @@
  */
 public class QueryManager {
 
-    private Map<String, String> queries = new HashMap<>();
+    private Map<String, String> queries;
 
     public QueryManager(String databaseType, String databaseVersion) throws
-            QueryMappingNotAvailableException, ConfigurationException, IOException {
+            QueryMappingNotAvailableException, ConfigurationException{
```
Comment (Anoukh) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/QueryManager.java]: Missing a space

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java [hunk 2]
```diff
@@ -513,100 +509,105 @@ public List selectAppAggOverallMetrics(String metricsType, String workerId, long
...
+    
     /**
      * Used to get the metrics gauges of jvm metrics.
      *
      * @param workerId       source ID of the metrics.
      * @param timeInterval   time interval that needed to be taken.
      * @param metricTypeName metrics type name ex: memory,cpu
      * @param currentTime    current time in milliseconds.
-     * @return List<List < Object>> of metrics data because charts needed in that format
+     * @return List<List< Object>> of metrics data because charts needed in that format
```
Comment (Anoukh) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java]: Extra space

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java [hunk 3]
```diff
@@ -513,100 +509,105 @@ public List selectAppAggOverallMetrics(String metricsType, String workerId, long
...
-
+    
     /**
      * Used to ge the overall throughput of the worker.
      *
      * @param workerId     source id of the metrics
      * @param timeInterval time interval that metrics needed to be taken.
      * @param currentTime  current time
-     * @return List<List       <       Object>> of metrics data because charts needed in that format
+     * @return List<List   <   Object>> of metrics data because charts needed in that format
```
Comment (Anoukh) [inline components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java]: Extra spaces

Comment (Yasara123) [reply components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java]: Fixed in 3fb0153

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java (hunk 2 above)
Comment (Yasara123) [reply components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardMetricsDBHandler.java]: Fixed in 3fb0153

File: components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/QueryManager.java (hunk 1 above)
Comment (Yasara123) [reply components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/QueryManager.java]: Fixed in 3fb0153

File: components/org.wso2.carbon.siddhi.metrics.core/src/test/java/org/wso2/carbon/siddhi/metrics/core/StatisticsTestCase.java [hunk 4]
```diff
@@ -158,7 +163,6 @@ public void receive(Event[] events) {
             inputHandler.send(new Object[]{"IBM", 75.6f, 100});
         }
         Thread.sleep(3010);
```
Comment (Anoukh) [inline components/org.wso2.carbon.siddhi.metrics.core/src/test/java/org/wso2/carbon/siddhi/metrics/core/StatisticsTestCase.java]: Can we use awaitlity here?

Comment (Yasara123) [reply components/org.wso2.carbon.siddhi.metrics.core/src/test/java/org/wso2/carbon/siddhi/metrics/core/StatisticsTestCase.java]: Fixed in a119095

**Pull request merged**